applications/tari_dan_wallet_web_ui/src/routes/AssetVault/Components/SendMoney.tsx

```diff
@@ -189,7 +189,10 @@ export function SendMoneyDialog(props: SendMoneyDialogProps) {
             });
             return;
           }
-          setTransferFormState({ ...transferFormState, fee: result.fee.toString() });
+          // temporary fix for having a proper calculation for estimated fee to avoid not enough fee issue
+          // TODO: remove once fixed in backend
+          const fee = (result.fee += 100);
```

Comment:
nit: no need to mutate result type
```suggestion
          const fee = result.fee + 100;
```